CHANGELOG.md

```diff
@@ -11,65 +11,65 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.2.0] - 2020-04-20
 
 ### Added
 
-- Support of IEEE files [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/ieee14.gif)
-- Creation of a main apps menu [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/apps-menu.gif)
-- Enable Pan/Zoom in a substation window [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/pan-substation-view.gif)
-- Center the map on a substation on user clicks [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/center-map.gif)
-- All/none actions for the voltage levels on the map [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/all-none.gif)
+-   Support of IEEE files [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/ieee14.gif)
+-   Creation of a main apps menu [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/apps-menu.gif)
+-   Enable Pan/Zoom in a substation window [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/pan-substation-view.gif)
+-   Center the map on a substation on user clicks [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/center-map.gif)
+-   All/none actions for the voltage levels on the map [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/all-none.gif)
 
 ### Changed
 
-- New fancy study-add button [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/add-button.gif)
-- _undefined_ is no more displayed in the substation list when country info is unavailable [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/country-info.gif)
+-   New fancy study-add button [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/add-button.gif)
+-   _undefined_ is no more displayed in the substation list when country info is unavailable [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/country-info.gif)
 
 ### Fixed
 
-- Fix authentication when reloading [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/fix-refresh.gif)
+-   Fix authentication when reloading [[feature demo]](https://gridsuite.github.io/demo/v0.2.0/fix-refresh.gif)
 
 ## [0.1.0] - 2020-04-06
 
 ### Added
 
-- This new CHANGELOG file can be used as a communication support for the users concerning our main exposed functionalities
-- Manage authentication from an Azure account
-- Create and visualize studies from UCT, IIDM and CGMES network files
-- Geographical display of the imported networks
-- On-demand display of electrical substations
+-   This new CHANGELOG file can be used as a communication support for the users concerning our main exposed functionalities
+-   Manage authentication from an Azure account
+-   Create and visualize studies from UCT, IIDM and CGMES network files
+-   Geographical display of the imported networks
+-   On-demand display of electrical substations
 
 [unreleased]: https://github.com/gridsuite/study-app/compare/v0.4.0...HEAD
 [0.4.0]: https://github.com/gridsuite/study-app/releases/tag/v0.4.0
```

Comment:
https://github.com/gridsuite/gridstudy/releases/tag/v0.4.0 ?
...

Reply:
I had a rebase problem, I'm fixing it :+1: